server/views/partials/personalPage/distinguishingMarks.njk

```diff
@@ -0,0 +1,66 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% set groupedDistinguishingMarks = distinguishingMarks | groupDistinguishingMarks %}
+{% set highlightConfig = groupedDistinguishingMarks.highlights %}
+
+{% macro identifyingMark(info) %}
+    {% if info and info | length %}
+        {% for bodyPart, marks in info %}
+            <details class="govuk-details">
+                <summary class="govuk-details__summary">
+                    <span class="govuk-details__summary-text">
+                        {{ bodyPart | capitalize }}
+                    </span>
+                </summary>
+                <div class="govuk-details__text">
+                    {% for mark in marks %}
+                        <dl>
+                            <div class="personal-distinguishing-marks__row__populated__row">
+                                <dt class="personal-distinguishing-marks__row__mark__row__key">Location</dt>
+                                <dd class='govuk-!-margin-left-0'>{{ mark.location }} </dd>
+                            </div>
+
+                            <div class="personal-distinguishing-marks__row__mark__row">
+                                <dt class="personal-distinguishing-marks__row__mark__row__key">Description</dt>
+                                <dd class='govuk-!-margin-left-0'>{{ mark.comment or "Not entered" }}</dd>
+                            </div>
+                        </dl>
+                    {% endfor %}
+                </div>
+            </details>
+        {% endfor %}
+    {%  else %}
+        Not entered
+    {% endif %}
+{% endmacro %}
+
+<h3 class="govuk-heading-m">Distinguishing marks</h3>
+<div class='personal-distinguishing-marks'>
+    <div class='personal-distinguishing-marks__image-container govuk-!-padding-left-3 govuk-!-padding-right-8'>
+        <img src='/assets/images/distinguishingMarks/base-person.svg' class='dm-base-image' width='300' height='572' alt=""/>
```

Comment:
Original plan for accessibility was to add an aria-describedby to a div and build up some text based on what is highlighted. However, as this is pretty much decorative and all the information is contained in the actual text, I think just hiding it is probably better. Open to discussion